packages/web/src/pages/upload-page/fields/TrackMetadataFields.tsx

```diff
@@ -56,6 +56,7 @@ export const TrackMetadataFields = (props: TrackMetadataFieldsProps) => {
             aria-label={messages.genre}
             placeholder={messages.genre}
             mount='parent'
+            // TODO: Use correct value for Genres based on label (see `convertGenreLabelToValue`)
```

Comment:
why is this TODO?

Reply:
it's new upload flow, see other comment

Reply:
jk i think i figured it out @raymondjacobson 
@amendelsohn can you ptal :D